content/en/tracing/trace_pipeline/ingestion_controls.md

```diff
@@ -136,7 +136,7 @@ To specify a specific percentage of a service's traffic to be sent, add an envir
 1. Select the service you want to change the ingested span percent for.
 2. Choose the service language.
 3. Choose the desired ingestion percentage.
-4. Apply the appropriate configuration generated from these choices to the indicated service and redeploy the service.
+4. Apply the appropriate configuration generated from these choices to the indicated service and redeploy the service. **Note**: The service name value is case sensitive. It should match the case of your service name.
```

Comment:
Did you want this note on a new line?

Reply:
nah, it's not that important most of the time, to take up that whitespace